src/configs/ToastConfig.js

```diff
@@ -18,12 +17,12 @@ export const toastConfig = {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.white,
+    backgroundColor: 'white',
     alignItems: 'center',
     justifyContent: 'center',
     padding: 10,
     borderRadius: 10,
-    borderColor: COLORS.concrete,
+    borderColor: '#f2f2f2',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values may not be consistent across the application. It's better to use a centralized color configuration for maintainability and consistency.
```diff
-    backgroundColor: 'white',
+    backgroundColor: colors.white,
...
-    borderColor: '#f2f2f2',
+    borderColor: colors.concrete,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->